### PR TITLE
Update Swatches

### PR DESCRIPTION
**Pull request by dimitrisnl**
Fixes: #872
Change-type: minor
Signed-off-by: Dimitrios Lytras <dimitrios@balena.io>

---
##### Contributor checklist

- [ ] I have rebuilt the README with `npm run build:docs`
- [ ] I have regenerated screenshots for any affected components with `npm run generate-screenshots`
- [ ] I have regenerated jest snapshots for any affected components with `npm run jest -- -u`

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

**Comment by dimitrisnl** [comment]
I notice that `quartenary` `dark` version has the "to be determined" label.

**Comment by sradevski** [comment]
@dimitrisnl I actually was working on this before I went to lunch, we should have synced :)

**Comment by sradevski** [comment]
@dimitrisnl another thing, no wait for the dark quartenary to be decided, I would prefer if we just submit a fix PR when it is actually used and decided.

File: src/theme.ts [hunk 1]
```diff
@@ -182,9 +173,7 @@ export default {
 			},
 			zIndex: 45,
 			// TODO: Border color should be tertiary.main once the TODOs in the Select component are resolved.
-			extend: `color: ${colors.secondary.main}; border: 1px solid ${
```
Comment (sradevski) [inline src/theme.ts]: There are a bunch of places where we use `quartenary.dark` but now it should be `quartenary.main`.

Comment (dimitrisnl) [reply src/theme.ts]: I see thanks @sradevski 	Will update them.
Are you approaching this in a different manner? Want to merge our changes or close this PR?

Comment (sradevski) [reply src/theme.ts]: No, I just reverted the theme change and added `quartenary` to the storybooks, so you can go ahead with this and I will separately submit a PR.

**Comment by dimitrisnl** [comment]
@balena-ci 	retest

**Pull request merged**